src/persistence.ts

```diff
@@ -48,22 +48,22 @@ export interface ConditionalDeleteResourceRequest {
 
 export interface InitiateExportRequest {
     requesterUserId: string;
-    requestGranularity: ExportRequestGranularity;
+    requestGranularity: ExportType;
     transactionTime: number;
-    requestQueryParams?: { _outputFormat?: string; _since?: number; _type?: string };
+    outputFormat?: string;
+    since?: number;
+    type?: string;
     groupId?: string;
 }
 
 export interface GetExportStatusResponse {
     jobStatus: string;
     exportedFileUrls?: [{ type: R4Resource; url: string }];

```

Comment:
type should just be a string. if we think of Profile support there will be types not in R4 or STU3

Reply:
fixed